spec.md

```diff
@@ -2650,6 +2661,17 @@ Integer(42)
 Integer(-42)
 ```
 
+### List Property Literal Expresssions
+
+A literal `List`{.firrtl} property type expression can be created from a
+sequence of property values of the element type.  The following examples show
+literal `List`{.firrtl} property type expressions.
+
+```firrtl
+List<Integer>(Integer(42), Integer(0))
+List<List<Integer>>(List<Integer>(), List<Integer>(Integer(5)))
+```
+
```

Comment:
We should probably use the term "value" instead of "literal" since you can have a List of Integer properties that are not literals, eg.
```
input foo : Integer
output bar : List<Integer>
propassign bar, List<Integer>(foo, Integer(42))
```

Reply:
Haha, okay so I'm happy either way!  Had same doubt and was gently encouraged to keep with literal terminology.

Whatever answer we like best lets be consistent throughout the stack :+1: .

If not a literal will need to tweak the structure a bit but that seems fine and we'll surely want other sorts of expressions at some point.  Will circle back, regardless thanks for the attention to detail and bringing this up!

Reply:
Totally, that's the right phrasing.  Mostly blocked on either reworking this section to not be called "Property Literal Expressions" or split it out, words are hard.   Anyway, just wanted to note lack of progress is not lingering disagreement :+1:, just need some free writing cycles.